plugin-server/src/main/graphile-worker/graphile-worker.ts

```diff
@@ -189,20 +212,23 @@ export class GraphileWorker {
         })
     }
 
-    async start(jobHandlers: TaskList): Promise<void> {
+    async start(jobHandlers: TaskList, crontab: CronItem[] = []): Promise<void> {
```

Comment:
non-blocking

Given that in reality I don't think we'd ever want to change jobHandlers or crontab, it would be nice if GraphileWorker properties were immutable i.e. passing them into the constructor and only setting them there. Not important though but might avoid potential mutability bugs down.

Reply:
Yeah I agree - this is part of the legacy architecture that uses `start` rather than relying on the constructor. I need to look into this and will refactor it